<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hotPlace</class>
 <widget class="QWidget" name="hotPlace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MetroPlus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/image/logo.png</normaloff>:/new/prefix1/image/logo.png</iconset>
  </property>
  <widget class="QLabel" name="menuBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(240, 240, 240,150);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="menuWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>171</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>172</width>
      <height>263</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="menu">
     <item>
      <widget class="QPushButton" name="travel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;
color: rgb(136, 136, 136);
background:transparent;
border-width:0;
border-style:outset;
text-align : left;</string>
       </property>
       <property name="text">
        <string>出行</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="1" column="1">
        <widget class="QPushButton" name="realTime">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>实时地铁     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/realTimeIcon.png</normaloff>:/new/prefix1/image/realTimeIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="historyRoute">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>历史行程     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/operationTimeIcon.png</normaloff>:/new/prefix1/image/operationTimeIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="operationTime">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>运营时间     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/historyRouteIcon.png</normaloff>:/new/prefix1/image/historyRouteIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="routePlanning">
         <property name="styleSheet">
          <string notr="true">QPushButton { 
background:transparent;
border-width:0;
border-style:outset;
font: 13pt &quot;微软雅黑&quot;;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
background-color: rgba(0,0,0,20);
border-radius: 8px;
}
</string>
         </property>
         <property name="text">
          <string>路线规划     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/routePlanningIcon.png</normaloff>:/new/prefix1/image/routePlanningIcon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="recommend">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;
color: rgb(136, 136, 136);
background:transparent;
border-width:0;
border-style:outset;
text-align : left;</string>
       </property>
       <property name="text">
        <string>推荐</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="1">
        <widget class="QPushButton" name="hotPlace_2">
         <property name="styleSheet">
          <string notr="true">font: 13pt Microsoft YaHei UI;color: rgb(255, 255, 255);border-radius: 8px;background-color: rgba(0,0,0, 70);</string>
         </property>
         <property name="text">
          <string>热门地点     </string>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/new/prefix1/image/hotPlaceIcon_active.png</normaloff>:/new/prefix1/image/hotPlaceIcon_active.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="person">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;微软雅黑&quot;;
color: rgb(136, 136, 136);
background:transparent;
border-width:0;
border-style:outset;
text-align : left;</string>
       </property>
       <property name="text">
        <string>我的</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/LOGO2.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
background:transparent;
font: 12pt &quot;黑体&quot;;
border-style:outset;
color: rgba(0, 0, 0, 150);
 } 

QPushButton:hover { 
color: rgba(0, 0, 0, 200);
}
</string>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/exit_image.png</normaloff>:/new/prefix1/image/exit_image.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 12pt &quot;黑体&quot;;
border-style:outset;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profile_photo">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-radius: 15px;
border-image: url(:/new/prefix1/image/head1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="cloth_button">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/cloth_image.png</normaloff>:/new/prefix1/image/cloth_image.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 600 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>热门景点</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>951</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-521</y>
      <width>939</width>
      <height>1020</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1020</height>
     </size>
    </property>
    <widget class="QLabel" name="hp1">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>1</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p4.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp5">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>0</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p5.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp9">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>260</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p9.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp8">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>260</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p8.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp6">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>261</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p6.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>260</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p7.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp10">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>260</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p10.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp13">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>519</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p13.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp12">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>519</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p12.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp14">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>520</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p14.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp11">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>520</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p11.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp15">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>520</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p15.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp17">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>779</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p17.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp18">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>780</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p18.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp19">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>779</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p19.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp16">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>780</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p16.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hp20">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>779</y>
       <width>180</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/p20.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="title_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>630</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 600 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>商圈购物</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>660</y>
     <width>951</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-241</y>
      <width>939</width>
      <height>420</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>420</height>
     </size>
    </property>
    <widget class="QLabel" name="sh1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sh2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sh4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>110</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh4.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sh3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh3.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sh6">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>220</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh6.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sh5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh5.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sh7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh7.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sh8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>330</y>
       <width>451</width>
       <height>85</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/image/sh8.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
